src/org/zaproxy/zap/extension/pscanrulesBeta/CSRFCountermeasures.java

```diff
@@ -104,15 +104,14 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 		
 		long start = System.currentTimeMillis();
 		
-		ExtensionAntiCSRF extAntiCSRF = 
-			(ExtensionAntiCSRF) Control.getSingleton().getExtensionLoader().getExtension(ExtensionAntiCSRF.NAME);
+		ExtensionAntiCSRF extAntiCSRF = getExtensonAntiCSRF();
 		
 		if (extAntiCSRF == null) {
 			return;
 		}
 		
-		List<Element> formElements = source.getAllElements(HTMLElementName.FORM);
 		List<String> tokenNames = extAntiCSRF.getAntiCsrfTokenNames();
+		List<Element> formElements = source.getAllElements(HTMLElementName.FORM);
```

Comment:
Not needed. I think the lines where unintentionally swapped. Only the order changed. Want me to revert this?